cloudinit/config/cc_ansible.py

```diff
@@ -196,3 +216,15 @@ def run_ansible_pull(pull: AnsiblePull, cfg: dict):
     )
     if stdout:
         sys.stdout.write(f"{stdout}")
+
+
+def ansible_galaxy(cfg: dict, ansible: AnsiblePull):
+    actions = cfg.get("actions", [])
+    ansible_config = cfg.get(CFG_OVERRIDE, "")
+    if ansible_config:
+        ansible.env[CFG_OVERRIDE] = ansible_config
+
+    if not actions:
+        LOG.warning("Invalid config: %s", cfg)
+    for command in actions:
+        ansible.subp(command)
```

Comment:
> Otherwise, a user could insert any arbitrary command here

In general, I would argue that if config writers (who have superuser on their machine) want to `rm -rf /etc`, I don't see a reason to stop them. They can already do that in `bootcmd` / `runcmd` if they would like to.

That said, this is a specialized command where running commands besides `ansible-galaxy` doesn't make sense. If `ansible-galaxy` had names on other distros, or different names based which install type it had, I'd say that would be a fairly strong reason to leave the flexibility as is.

I'm fairly indifferent, I can make the suggested change.

Reply:
It was an open question, maybe exacerbated by the leaking to the logs' discussion, ignore my comment please.